tests/contest/contest/src/tests/cgroups/cpu/v1.rs

```diff
@@ -1,12 +1,16 @@
+use std::fs;
 use std::path::Path;
+use std::string::ToString;
 
+use anyhow::{Context, Result};
```

Comment:
Do we need to add this here? The existing tests used normal `Result` so should keep new one the same if possible.

Reply:
I've remove `Context`. However, I believe it's necessary to use `Result` if we would like to check with `test_result` macro. Since there is already lots of functions like `create_cpu_spec` returning `anyhow::Result`, I guess it's okay to import `Result` in v1.rs.

Reply:
ok, works :+1: I was only concerned as the tests were working without the imports before, and didn't wanted to add once which are not strictly needed.